hledger-lib/hledger-lib.cabal

```diff
@@ -110,6 +110,7 @@ library
       Hledger.Data.Dates
       Hledger.Data.Journal
       Hledger.Data.Ledger
+      Hledger.Data.MarketPrice
```

Comment:
As this is generated from package.yaml, add the module there too. This will probably fix the Travis error.